cypress/e2e/functional/document_tests/tiles_copy_test_spec.js

```diff
@@ -292,6 +292,38 @@ context('Test copy tiles from one document to other document', function () {
     testPrimaryWorkspace2();
 
   });
+  it('should enable/disable copy-to-workspace and copy-to-document buttons appropriately', function () {
+    beforeTest(student5);
+    cy.openTopTab('problems');
+    cy.openProblemSection('Initial Challenge');
+
+    // Verify initial disabled state and no tiles selected
+    canvas.getCopyButtons().should('have.class', 'disabled');
```

Comment:
I'd add `.should('not.have.class', 'enabled')` in addition everywhere you have `should('have.class', 'disabled')`,  and add  `.should('not.have.class', 'disabled')` everywhere you have `should('have.class', 'enabled')`